pkg/server/testserver.go

```diff
@@ -353,7 +353,7 @@ func (ts *TestServer) ExpectedInitialRangeCount() (int, error) {
 // assuming no additional information is added outside of the normal bootstrap
 // process.
 func (ts *TestServer) ExpectedInitialUserRangeCount() int {
-	return ExpectedInitialUserRangeCount(ts.DB())
+	return 1
```

Comment:
Looks like you could unravel this thread more, if you feel so inclined.

Reply:
Done.